static/app/components/group/assignedTo.tsx

```diff
@@ -184,6 +192,40 @@ function AssignedTo({
     }
   );
 
+  const {mutate: handleAssigneeChange} = useMutation<
+    AssignableEntity | null,
+    RequestError,
+    AssignableEntity | null
+  >({
+    mutationFn: async (
+      newAssignee: AssignableEntity | null
+    ): Promise<AssignableEntity | null> => {
+      setAssigneeLoading(true);
+      if (newAssignee) {
+        await assignToActor({
+          id: group.id,
+          orgSlug: organization.slug,
+          actor: {id: newAssignee.id, type: newAssignee.type},
+          assignedBy: 'assignee_selector',
+        });
+        return Promise.resolve(newAssignee);
+      }
+
+      await clearAssignment(group.id, organization.slug, 'assignee_selector');
+      return Promise.resolve(null);
+    },
+    onSuccess: (newAssignee: AssignableEntity | null) => {
+      if (onAssign && newAssignee) {
+        onAssign(newAssignee.type, newAssignee.assignee, newAssignee.suggestedAssignee);
+      }
+      setAssigneeLoading(false);
```

Comment:
No need for a separate loading state in this component, `useMuatation` returns `isLoading` for this purpose

Reply:
So I was going to say that I tried this when I replaced the assignee dropdown in issues stream and it refused to work... but then I went to record an instance of it not working in issue details, and it works now??